### DISCUSSION TITLE
딥다이브 주제 뭘 해야할지 모르겠습니다.

### DISCUSSION THREAD

**Discussion by lky3004me** [100-hours-a-week/discussions#67] [개발] [answered]
1. 분산 서버를 활용한 크롤링 성능 최적화
2. 셀러리와 래빗 mq를 활용한 크롤링 분산 처리와 이를 통한 성능 증가
3. virtual thread 적용을 통한 io bound 시간 감소를 통한 성능 증가

위 3개가 제가 생각했던 딥다이브 주제입니다.
저희 프로젝트의 주제가 최저가 탐색이고, 제가 크롤링을 담당하고 있는 만큼 크롤링과 관련이 있으면서 성능 증가에 유의미한 결과를 보이리라 기대한 것이 위 3개입니다. 

다만 예전에 제프가 말씀해주셨다시피, 병렬처리가 지나치게 범위가 넓은 항목이라 면접관에게 오히려 좋지 못하게 보일 여지가 크다고 말씀해주셨습니다. 

또한 고민인 점은 제가 이번 교육 과정 기간을 제외하고 파이썬과 크롤링을 자주 사용하지는 않았다는 점입니다. 이로 인해 파이썬과 크롤링 관련 주제를 더 깊이 파고드는 것이 제 커리어 형성에 도도움이 될지 궁금한 상황입니다. 

혹시 프로젝트 관련 주제를 추천해주시거나 방향을 지정해주신다면 감사할 것 같습니다. 또한 이전 주제를 검토하셔서 더 나은 방향이 있다면 조언 부탁드립니다!

**Comment by ej31** [comment, marked as answer]
# 답변
- 이건 만나서 얘기해봅시다!
